### DISCUSSION TITLE
서버수 급락 삭제 요청

### DISCUSSION THREAD

**Discussion by samsunghappytree123** [despenser08/KRList-Stats#89] [Delete Request] [answered]
https://koreanbots.dev/bots/789452453401722920

서버수 급락 부분만 지워주시면 감사하겠습니다.

**Comment by despenser08** [comment, marked as answer]
처리가 완료되었습니다.
![image](https://user-images.githubusercontent.com/50207925/135723403-f3b55106-8093-4ea8-968d-1eb1170986ab.png)
